documentation/rfc/distributed.md

```diff
@@ -0,0 +1,36 @@
+Scenario Distributed
+====================
+
+
+This scenario allow us to call multiple engines for computing fallbacks. These engines can be:
+
+  - kraken
+  - geovelo
+  - here
+  - valhalla
+
+The orchestration is a bit complex, so this documentation will try to explain what is done, and in what order.
+Each fallback mode can use a specific engine that is configured at the instance level.
+
+First we search for the origin and destination objects with kraken.
+
+Then we compute for each departure fallback modes the direct path between the origin and the destination with the
+appropriate engine. This is done asynchronously.
+If the public transport is disabled (max_duration=0) jormungandr return these results and stop.
+
+Then we start computing the stop points that can be reached for each mode at both extremity. 
+This is done by using a crowfly at constant speed that is computed by kraken.
+
+While this is computing we look for stop point that can be accessed in zero sec (typically the stop_point
+of the stop_area). This might require a call to kraken in some case.
+
+At this point we have to wait for all of the previous task to finish, when this is done we can create a matrix of
+stop_points accessible per mode, that we update with "real" duration by using the configured engine for each mode.
+This is where we use the matrix capabilities of the streetnetwork engines.
```

Comment:
That's not true, there is no join at this point. When the 2 needed streetnetwork and the direct path for a kraken request is ready, the pt computation is launched.

Reply:
that's what I meant to say, but not what wrote :/